code/controllers/subsystem/air.dm

```diff
@@ -244,8 +249,18 @@ var/datum/subsystem/air/SSair
 		if(T.excited_group)
 			T.excited_group.garbage_collect()
 
+/datum/subsystem/air/proc/activate_queued_turfs()
+	if(LAZYLEN(to_activate))
+		for(var/T in to_activate)
+			add_to_active(T, to_activate[T])
+	to_activate = null
```

Comment:
This isn't actually needed because turfs check if they need to be in the active list when ssair runs setup_allturfs()